app/models/role.rb

```diff
@@ -0,0 +1,14 @@
+class Role < ApplicationRecord
```

Comment:
We can introduce an `enum` in the role class which will specify all the roles our app will support, so that no one can create any other roles except we define as enums.

`enum name: {admin: 'admin', author: 'author', reader: 'reader'}`